nestkernel/nest_names.cpp

```diff
@@ -37,6 +37,10 @@ const Name Act_m( "Act_m" );
 const Name address( "address" );
 const Name alpha_1( "alpha_1" );
 const Name alpha_2( "alpha_2" );
+const Name AMPA_E_rev( "AMPA_E_rev" );
```

Comment:
Why start the names with ``AMPA``? ``E_rev_AMPA`` would look more in accordance with the naming standards to me... same applies for the others.